package.json

```diff
@@ -28,7 +28,8 @@
     "sense": "nebula sense --ext src/ext.js --meta src/meta.json",
     "test": "echo No tests found...",
     "test:integration": "aw puppet --testExt '*.int.js' --glob 'test/integration/**/*.int.js'",
-    "release": "release-it"
+    "release": "release-it",
+    "link:nebula": "yarn link '@nebula.js/cli-serve' && echo 'To use react from nebula replace react and react-dom version number \nin package.json e.g. 18.2.0 with:' && echo '\"file:\\c' && (readlink -f ./node_modules/@nebula.js/cli-serve | sed -r 's/\\/commands\\/serve/\\/node_modules\\/react\"/g') && echo '\"file:\\c' && (readlink -f ./node_modules/@nebula.js/cli-serve | sed -r 's/\\/commands\\/serve/\\/node_modules\\/react-dom\"/g')"
```

Comment:
I guess this will be while developing only. We should try to find a proper solution with `peerDependencies` later on.

Reply:
Yes, I created an issue: https://github.com/qlik-oss/sn-filter-pane/issues/14